share/revng/test/configuration/revng-qa/architectures.yml

```diff
@@ -87,11 +98,28 @@ tags:
       QEMU_NAME: s390x
       COMMON_CFLAGS:
         - -DTARGET_s390x
+      CLANG_CFLAGS:
+        - --target=systemz
   - name: aarch64
     implies: [cross-compiler]
     variables:
       TRIPLE: aarch64-unknown-linux-musl-
       POINTER_SIZE: "64"
       QEMU_NAME: aarch64
+      APPLE_NAME: arm64
       COMMON_CFLAGS:
         - -DTARGET_aarch64
+      CLANG_CFLAGS:
+        - --target=aarch64
+
+  - name: apple-x86-64
+    implies: [x86-64, clang-only]
+    variables:
+      CLANG_CFLAGS:
+        - --target=x86_64-apple-macos-macho
```

Comment:
Where did you get the triple from? I'd expect `darwin` somewhere.

Reply:
By fiddling with it until it worked. As far I understand, the only parts we care about are `x86_64` (specifically with underscore), `apple` and `macho`. Everything else doesn't seem to have meaningful impact onto anything I have seen. I did keep `macos` just in case though.

Reply:
Let's refrain from making up new triple, they make me cry.
Use one of these:
```
.orchestra/config/components/toolchain/macos/clang.yml:x86-64: x86_64-apple-darwin11
.orchestra/config/components/toolchain/macos/clang.yml:i686: i686-apple-darwin11
.orchestra/config/components/toolchain/macos/ld64.yml:x86-64: x86_64-apple-darwin11
.orchestra/config/components/toolchain/macos/ld64.yml:i686: i686-apple-darwin11
.orchestra/config/components/toolchain/macos/ld64.yml:arm: arm-apple-darwin11
.orchestra/config/components/toolchain/macos/ld64.yml:aarch64: aarch64-apple-darwin11
```

Reply:
These seem to work :tada: